### PR TITLE
global: renaming of example folder to examples 

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: src/invenio-records-js/templates/error.html [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<div ng-show="recordsVM.invenioRecordsError.data.message" class="alert alert-danger">
```
Comment (jirikuncar) [inline src/invenio-records-js/templates/error.html]: @drjova is this an intended change?

Comment (drjova) [inline src/invenio-records-js/templates/error.html]: yes, we use ``invenio-records-alert`` with ``alert.html`` template

Comment (jirikuncar) [inline src/invenio-records-js/templates/error.html]: Ahh ok. But shouldn't this change be a separate commit?

**Comment by jirikuncar** [comment]
@drjova `git commit --amend` & `s/rename/renaming/`
